content/product/iac.md

```diff
@@ -0,0 +1,241 @@
+---
+title: "Pulumi IaC: Infrastructure as Code"
```

Comment:
should the path be /infrastructure-as-code

and then have /iac redirect to it?

just want to make sure the path is SEO optimized

Reply:
We don't have an IaC page rn. So let's just change it to 'product/infrastructure-as-code/'. There should be no SEO hit for that.